mousestyles/GLRT_distribution.py

```diff
@@ -0,0 +1,211 @@
+from __future__ import print_function, absolute_import, division
+
+import numpy as np
+from mousestyles import data
+
+
+def powerlaw_pdf(x, a):
+    """
+    The probability density function of truncated power law.
+
+    Parameters
+    ----------
+    x : int
+        x in formula p(x) = (alpha-1)*x^(-alpha).
+    a : int>1
+        alpha in formula p(x) = (alpha-1)*x^(-alpha).
+
+    Returns
+    -------
+    probability density : a float number
+        The probability density of power law at x.
+
+    Examples
+    --------
+    >>> powerlaw_pdf (2,2)
+    0.25
+    """
+    return((a - 1) * x**(-a))
+
+
+def exp_pdf(x, l):
+    """
+    The probability density function of truncated exponential.
+
+    Parameters
+    ----------
+    x : int
+        x in formula p(x) = lambda*exp(-lambda*x).
+    l : int
+        lambda in formula p(x) = lambda*exp(-lambda*x).
+
+    Returns
+    -------
+    probability density : a float number
+        The probability density of power law at x.
+
+    Examples
+    --------
+    >>> exp_pdf
```

Comment:
Thank you for your advice. Yes, there is a typo there and I have already fixed it. Thank you again.